src/garage/torch/policies/base.py

```diff
@@ -0,0 +1,53 @@
+
+
+class Policy:
+    """
+    Policy base class without Parameterzied.
+
+    Args:
+        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+
+    """
+    def __init__(self, env_spec):
+        self._env_spec = env_spec
+        self._variable_scope = None
+
+    # Should be implemented by all policies
+    def get_action(self, observation):
+        """Get action given observation."""
+        raise NotImplementedError
+
+    def get_actions(self, observations):
```

Comment:
Why not define this to just call `get_action` on each sub-array?
That way it will work, even on non-vectorized policies.

Reply:
Oh. Ok. I will change the code

Reply:
I meant this:
```python
return np.stack([self.get_action(obs) for obs in observations])
```

Reply:
ok thanks